CMakeLists.txt

```diff
@@ -95,9 +95,13 @@ if (WITH_TITAN_TESTS OR WITH_TITAN_TOOLS)
     set(CMAKE_REQUIRED_FLAGS "--std=c++11")
     CHECK_CXX_SOURCE_COMPILES("
 #include <atomic>
-std::atomic<uint64_t> x(0);
+struct A { 
```

Comment:
why change

Reply:
This was for conditionally linking `libatomic`,  atomic ops over structs larger than 16 bytes require the bin link with `libatomic`. Turns out its not a easy thing to do in `rust-rocksdb`. I now use `mutex` over lock free, changing config is not a frequent op and in the code it already acquire the mutex anyway.

Reply:
https://gcc.gnu.org/onlinedocs/gcc/_005f_005fatomic-Builtins.html More readings about why we need libatomic.